packages/chains/src/mode/redemptionStrategies.ts

```diff
@@ -1,5 +1,48 @@
-import { assetSymbols, RedemptionStrategy, RedemptionStrategyContract, underlying } from "@ionicprotocol/types";
+import {assetSymbols, RedemptionStrategy, RedemptionStrategyContract, underlying} from "@ionicprotocol/types";
 
-const redemptionStrategies: RedemptionStrategy[] = [];
+import {assets} from "./assets";
+
+const redemptionStrategies: RedemptionStrategy[] = [
```

Comment:
@rhlsthrm please check if the liquidity of these pairs is acceptable; we might want to study the liquidity concentration in order to know better the LP rug pull risk
https://app.kim.exchange/pools